drivers/sensors/azoteq_iqs5xx.h

```diff
@@ -0,0 +1,161 @@
+// Copyright 2023 Dasky (@daskygit)
+// Copyright 2023 George Norton (@george-norton)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "i2c_master.h"
+#include "pointing_device.h"
+#include "util.h"
+
+typedef enum { UNKNOWN, IQS550 = 40, IQS525 = 52, IQS572 = 58 } azoteq_product_numbers_t;
+typedef enum { ACTIVE, IDLE_TOUCH, IDLE, LP1, LP2 } azoteq_charging_modes_t;
+
```

Comment:
```suggestion
typedef enum { 
    UNKNOWN, 
    IQS550 = 40, 
    IQS525 = 52, 
    IQS572 = 58,
} azoteq_product_numbers_t;
typedef enum { 
    ACTIVE, 
    IDLE_TOUCH, 
    IDLE, 
    LP1, 
    LP2,
} azoteq_charging_modes_t;

```

Also, might be worth making the names more azoteq specific, to prevent potential naming collisions in the future?

Reply:
done